src/meltano/devcontainer-feature.json

```diff
@@ -1,6 +1,6 @@
 {
     "id": "meltano",
-    "version": "2.0.17",
+    "version": "3.5",
```

Comment:
This property is feature's version (it's independent from meltano's one). Please change version to `2.1.0` as you introduce new properties